packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -123,7 +126,18 @@ class BlockListBlock extends Component {
 
 		return (
 			<>
-				{ displayToolbar && <FloatingToolbar /> }
+				{ displayToolbar &&
+					<FloatingToolbar>
+						<Toolbar rtl passedStyle={styles.floating}>
+							<ToolbarButton
```

Comment:
This one is using a `ToolbarButton` while #17471 is using a custom component that replicates the functionality of `IconButton`. Can we make these two consistent?

I think we could keep `FloatingToolbar` as the generic UI component and something like `BlockNavigationToolbar` that puts the up button and breadcrumbs in a floating toolbar.

Reply:
I agree that we could keep button and breadcrumbs in separate component and make buttons consistent.

I think the best way to achieve that is `merge` this PR if rest of the code is fine. Then sync breadcrumb branch and do the refactor you mentioned. WDYT ?